e2e_tests/customerToken_test.py

```diff
@@ -13,13 +13,15 @@ def test_create_token():
     response = client.customerTokens.create_token(request)
     assert response.data.type == "customerBearerToken"
 
+
 def test_create_jwt_token():
     account_id = create_individual_customer()
     request = CreateCustomerToken(account_id, "customers accounts",
-                                  jwt_subject="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9fQ")
```

Comment:
Is `jwt_subject` and `jwt_token` the same field? What is the diff between these 2 fields?

Reply:
No, they are not the same, you understood correctly it should be the token.
the jwt_subject should be set for the customer in order to identify that the token sent is related to the specific customer